webapp/channels/src/components/admin_console/ldap_wizard/ldap_text_setting.tsx

```diff
@@ -75,8 +75,10 @@ const LDAPTextSetting = (props: TextSettingProps) => {
     const showFilterIcon = hasContent && props.filterResult != null; // loose equality operator is intentional
 
     // Determine icon type and content - three states
-    const isSuccess = props.filterResult?.error === '' && (props.filterResult?.total_count || 0) > 0;
-    const isWarning = props.filterResult?.error === '' && (props.filterResult?.total_count || 0) === 0;
+    const isFilter = isFilterTest(props.filterResult);
```

Comment:
Wondering if `boolean` is the optimal choice here, vs something slightly more extensible and self-descriptive, such as an enum. `!isFilter` means attributes check right?